JointMeasurements/Tests.qs

```diff
@@ -76,19 +105,19 @@ namespace Quantum.Kata.JointMeasurements {
     
     // ------------------------------------------------------
     operation T01_SingleQubitMeasurement_Test () : Unit {
-        DistinguishStates_MultiQubit(2, 2, StatePrep_ParityMeasurement, SingleQubitMeasurement, false);
+        DistinguishStates_MultiQubit(2, 2, StatePrep_ParityMeasurement, SingleQubitMeasurement, 0, ["(1/√2)|00⟩ + (1/√2)|11>", "(1/√2)|01⟩ + (1/√2)|10>"]);
```

Comment:
Notice that the task description promises *a superposition* of these basis states, not "an equal superposition" (specific superpositions can be distinguished and restored really easily). A better way to describe these states in a readable format is `["α|00⟩ + β|11⟩", ""α|01⟩ + β|10⟩"]`